src/components/standalone/security/ips/IpsSettings.vue

```diff
@@ -39,7 +53,90 @@ const options: Array<{
   }
 ]
 
-const ips = useIpsStore()
+const uciChangesStore = useUciPendingChangesStore()
+
+const loading = ref(true)
+const error = ref<Error>()
+const enabled = ref(false)
+const policy = ref<Policy>('connectivity')
+const oinkcode = ref('')
+
+function fetch() {
```

Comment:
Shall we move this function to `useIps` composable?

Reply:
Removed the `useIps` completely, we don't need it for now